src/compas_view2/objects/collectionobject.py

```diff
@@ -0,0 +1,9 @@
+from .meshobject import MeshObject
+
+
+class CollectionObject(MeshObject):
```

Comment:
why does the collection object inherit from the mesh object? what if you have a collection of meshes?

Reply:
Currently the mechanism is to join all shapes into a big mesh. So after init called, everything is the same with Mesh class. This class also works with Mesh, basically any class that has `to_vertices_and_faces`

Reply:
okay